website/docs/d/storage_account.html.markdown

```diff
@@ -35,13 +35,13 @@ output "storage_account_tier" {
 
 * `location` - The Azure location where the Storage Account exists
 
-* `account_kind` - Defines the Kind of account, either `BlobStorage` or `Storage`.
```

Comment:
#1316 raised the concern on missed option of StorageV2. Is it correct that we directly delete all the options in the documentation? Then where will user find these options? Shall we provide a link ?

Reply:
This is a datasource, not a resource so we don't to specify valid values. 

There is an argument to be said for listing possible values, however currently most of the documentation for datasources don't so for consistencies sake I have removed them.